src/Command/ConsumerCommand.php

```diff
@@ -46,6 +46,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getContainer();
         $gate = $input->getArgument('gate');
 
+        // retrieve the console path from the command line
+        $consolePath = $_SERVER['argv'][0] ?? 'bin/console';
```

Comment:
As I said in the comment : `realpath($_SERVER['argv'][0] ?? sprintf('%s/bin/console', $kernel->getRootDir()))`